test/common/pixel-tests

```diff
@@ -33,6 +38,20 @@ cmd_status() {
 }
 
 cmd_push() {
+    # Check if the current branch is behind the project's main and has newer pixel-tests, in this case
+    # the user has to rebase to not run into pixel-tests conflicts. There is no
+    # elegant way figuring out the real "origin" of the cockpit project so we
+    # discover the project's repository name and hardcode the upstream url.
+    project=$(git remote -v | awk '/^origin/ { print $2; exit }' | xargs basename)
+    git fetch https://github.com/cockpit-project/${project} main
+
+    cur_sha=$(get_pixel_reference "FETCH_HEAD")
+    main_sha=$(get_pixel_reference "HEAD")
```

Comment:
The names are mixed up, I think. `FETCH_HEAD` is the main branch, and `HEAD` is the current branch.

Reply:
Yes, indeed